src/main/arm/createUiDefinition.json

```diff
@@ -204,25 +204,47 @@
                         "visible": "[bool(basics('basicsOptional').basicsOptionalAcceptDefaults)]"
                     },
                     {
-						"name": "enableAdminHTTPListenPort",
-						"type": "Microsoft.Common.OptionsGroup",
-						"label": "Enable HTTP Listen Port on WebLogic Administration Server?",
-						"defaultValue": "Yes",
-						"toolTip": "Select 'No' to disable HTTP Listen Port on WebLogic Administration Server.",
-						"constraints": {
-							"allowedValues": [{
-									"label": "Yes",
-									"value": "true"
-								},
-								{
-									"label": "No",
-									"value": "false"
-								}
-							],
-							"required": true
-						},
+                        "name": "denyPublicTrafficForAdminServer",
+                        "type": "Microsoft.Common.OptionsGroup",
+                        "label": "Deny public traffic for admin server?",
+                        "defaultValue": "No",
+                        "toolTip": "Select 'Yes' to deny public traffic for admin server. Configuration here for port 7001 and 7002 has a higher priority than above.",
+                        "constraints": {
+                            "allowedValues": [
+                                {
+                                    "label": "No",
+                                    "value": false
+                                },
+                                {
+                                    "label": "Yes",
+                                    "value": true
+                                }
+                            ],
+                            "required": true
+                        },

```

Comment:
Hello @zhengchang907 I 100% approve of improving compliance with coding standards as we develop. However, in the future, separate such changes into separate PRs from changes that modify behavior.  For this PR, I'll happily accept it, but next time please do separate PRs.

Reply:
Sure I'll do it in separate PRs